src/config/chain/initia.ts

```diff
@@ -62,7 +62,8 @@ export const INITIA_CHAIN_CONFIGS: ChainConfigs = {
     wallets: [...keplrWallets],
     features: {
       faucet: {
-        enabled: false,
+        enabled: true,
+        url: "http://initia-faucet.alleslabs.dev",
```

Comment:
can we move inject this endpoint as a environment variable. not sure if exposing our faucet like this at the moment is okay

Reply:
done